File: old/fizzbuzz/src/pascal/fizzbuzz.pas
program fizzbuzz;
type
  range = 1..50;
var
  n : range;
begin
  for n := low(range) to high(range) do
    if (n mod 3 = 0) and (n mod 5 = 0) then
      writeln('fizz buzz')
    else if n mod 3 = 0 then
      writeln('fizz')
    else if n mod 5 = 0 then
      writeln('buzz')
    else
      writeln(n)
end.

File: old/fizzbuzz/src/fortran/fizzbuzz.f90
program fizzbuzz
  integer n
  do n = 1, 50
    if (mod(n, 3) == 0 .and. mod(n, 5) == 0) then
      print '(A)', 'fizz buzz'
    else if (mod(n, 3) == 0) then
      print '(A)', 'fizz'
    else if (mod(n, 5) == 0) then
      print '(A)', 'buzz'
    else
      print '(I0)', n
    end if
  end do
end program fizzbuzz
